### DISCUSSION TITLE
Are deleted media files auto-removed from playlists that contain them?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6229] [Q&A] [answered]
WIthout having to do lots of testing, assuming my media folder is a mounted NFS share and I delete several files, will AzuraCast automatically remove these deleted files from the playlist(s) that contain them? Or will I have to do that manually?

(My station is currently shuffling from 160988 songs and 38 playlists, it would be hard doing that manually…)

**Comment by BusterNeece** [comment, marked as answer]
Yes, it does that automatically.

**Comment by Moonbase59** [comment]
Good to know, thanks!
